### DISCUSSION TITLE
brew installl srccpy - Make brew find rust

### DISCUSSION THREAD

**Discussion by cassepipe** [Homebrew/discussions#2758] [Getting started]
I am trying to install srccpy on an Mac's friend (High Sierra).
Brew failed while building rust so I just installed it.
 ~/.cargo/bin is in my PATH.
But somehow brew does not seem be able to find and tries to compile it again.
Any idea of a fix ?
I suspect a basic command would do the trick but I am new to brew.

**Comment by SMillerDev** [comment]
Brew doesn't pick up random dependencies on the system. If something needs rust it needs the rust from brew.

**Reply by cassepipe** [reply]
Thank you.
So there's no workaround to brew failing to compile rust ?
Should I better try to troubleshoot the compilation failure ?

**Reply by carlocab** [reply]
Is your friend's `brew` installation up-to-date? IIRC there were fixes for compiling `rust` on older Macs made a little while back.
